apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/RpcRetryConsumer.cs

```diff
@@ -83,9 +83,9 @@ public void Receive(IEnumerable<T> items)
             {
                 _consumer.Receive(items);
             }
-            catch (RpcException e)
+            catch (RpcException e) when (!HandleException(items, e))
```

Comment:
This is reversed. Catch when `HandleException` is true and then don't throw in the catch block (same below).